scripts/get-tags

```diff
@@ -2,7 +2,7 @@
 maybe_get_tag () {
   if [ "$2" != "$3" ]
   then
-    echo "git tag -a $1-$2 -m ''"
+    echo "git tag -a $1_$2 -m ''"
```

Comment:
Caveat to the `sed` script is that we have to change to use `_` as the version prefix so we can replace it easily

Reply:
See above comment about changing the sed command